wdio.conf.js

```diff
@@ -21,7 +21,7 @@ exports.config = {
     // will be called from there.
     //
     specs: [
-        './test/OnlynerByTest.js'
+        './test/onlynerByTest.js'
     ],
```

Comment:
it will run usually one test `onlynerByTest`
We need to be able to run all tests in `test` folder